webapp/components/comments/CommentList/index.vue

```diff
@@ -0,0 +1,80 @@
+<template>
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-05-04T09:50:41Z" title="Saturday, May 4th 2019, 11:50:41 am +02:00">May 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,80 @@
+<template>
```


Nice, that you made an own component of this.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-05-04T12:23:03Z" title="Saturday, May 4th 2019, 2:23:03 pm +02:00">May 4, 2019</time>_

---


Thanks! did you see that one of the benefits of this, besides making it easier to test, is that I don't need to have a custom `CommentsByPost` resolver? I can just refetch the `Post` requesting just its comments, which have a subfield for its author?

Also, I learned how to call a sibling's method from another sibling, really cool stuff!!